javascript/organizing-js/es6-modules.md

```diff
@@ -158,7 +158,7 @@ This section contains questions for you to check your understanding of this less
 - <a class="knowledge-check-link" href="https://docs.npmjs.com/downloading-and-installing-packages-locally">Know how to install packages using npm.</a>
 - <a class="knowledge-check-link" href="https://peterxjang.com/blog/modern-javascript-explained-for-dinosaurs.html">Describe what a JavaScript module bundler like webpack is.</a>
 - <a class="knowledge-check-link" href="#webpack-knowledge-check">Explain what the concepts "entry" and "output" mean as relates to webpack.</a>
-- <a class="knowledge-check-link" href="https://webpack.js.org/concepts/dependency-graph/">Briefly explain what a development dependency is.</a>
+- <a class="knowledge-check-link" href="https://dev.to/moimikey/demystifying-devdependencies-and-dependencies-5ege">Briefly explain what a development dependency is.</a>
```

Comment:
Why is the new link's explanation "better" than the one provided by webpack?

Reply:
@marvingay Hi, sorry I forgot to explain why.

As a newbie, I think this source does a better job at explaining what a dev dependency is because it explains what dependencies in general are (like the webpack article, albeit it goes less deep into how these are bundled,) then marks a clear distintion between production and development dependencies (which the current source doesn't do.) It also provides examples of which type of dependencies are, or could be, dev dependencies, and how to save then in the config file.